q2_diversity/_beta/_method.py

```diff
@@ -110,19 +113,31 @@ def beta_phylogenetic_alt(table: BIOMV210Format, phylogeny: NewickFormat,
              variance_adjusted=variance_adjusted, bypass_tips=bypass_tips)
 
 
-def beta(table: biom.Table, metric: str, n_jobs: int=1)-> skbio.DistanceMatrix:
-    if metric not in non_phylogenetic_metrics():
+def aitchison(x, y, **kwds):
+    return euclidean(clr(x), clr(y))
```

Comment:
I know this is related to #213 but does the same problem arise if you compute the CLR transform for all the samples in the feature table and then individually call euclidean? i.e. if you change this function to:

```python
def aitchison(x, y, **kwds):
    return euclidean(x, y)
```